lib/ansible/modules/monitoring/zabbix/zabbix_user.py

```diff
@@ -0,0 +1,621 @@
+            usrgrps=dict(type='list', required=True),
+            passwd=dict(type='str', required=True, no_log=True),
+            override_passwd=dict(type='bool', required=False, default=False),
+            lang=dict(type='str', default='en_GB', choices=['en_GB', 'en_US', 'zh_CN', 'cs_CZ', 'fr_FR',
+                                                            'he_IL', 'it_IT', 'ko_KR', 'ja_JP', 'nb_NO',
+                                                            'pl_PL', 'pt_BR', 'pt_PT', 'ru_RU', 'sk_SK',
+                                                            'tr_TR', 'uk_UA']),
+            theme=dict(type='str', default='default', choices=['default', 'blue-theme', 'dark-theme']),
+            autologin=dict(type='bool', default=False),
+            autologout=dict(type='str', default='0'),
+            refresh=dict(type='str', default='30'),
+            rows_per_page=dict(type='str', default='50'),
+            after_login_url=dict(type='str', default=''),
+            user_medias=dict(type='list', required=True,
+                             elements='dict',
+                             options=dict(
+                                 mediatype=dict(type='str', default='Email'),
+                                 sendto=dict(type='str', required=True),
+                                 period=dict(type='str', default='1-7,00:00-24:00'),
+                                 severity=dict(type='dict',
+                                               options=dict(
+                                                   not_classified=dict(type='bool', default=True),
+                                                   information=dict(type='bool', default=True),
+                                                   warning=dict(type='bool', default=True),
+                                                   average=dict(type='bool', default=True),
+                                                   high=dict(type='bool', default=True),
+                                                   disaster=dict(type='bool', default=True)),
+                                               default=dict(
+                                                   not_classified=True,
+                                                   information=True,
+                                                   warning=True,
+                                                   average=True,
+                                                   high=True,
+                                                   disaster=True
+                                               )),
+                                 active=dict(type='bool', default=True)
+                             )),
+            type=dict(type='str', default='Zabbix user', choices=['Zabbix user', 'Zabbix admin', 'Zabbix super admin']),
+            state=dict(type='str', default="present", choices=['present', 'absent']),
+            timeout=dict(type='int', default=10)
+        ),
+        supports_check_mode=True
```

Comment:
Here it says that the module supports check mode, but it does not.
Can you please add a support for the check mode or set `supports_check_mode` o `False` for now?

Reply:
Good catch, I would prefer that this gets implemented rather than setting `supports_check_mode` to `False`, but either way, it must be fixed.

Reply:
Thanks for reviewing this @rubentsirunyan

Reply:
Fix to be able to compare parameters.
For example, the following is an example when lang is changed.

```
# ANSIBLE_PYTHON_INTERPRETER=venv/bin/python ansible-playbook -i inventory main.yml --diff --check

PLAY [zabbix_user module test] ***************************************************************************************************************************************************

TASK [update zabbix user params.] *****************************************************************************************************************************
--- before
+++ after
@@ -2,7 +2,7 @@
     "alias": "example",
     "autologin": "0",
     "autologout": "0",
-    "lang": "ja_JP",
+    "lang": "en_GB",
     "name": "user name2",
     "refresh": "30",
     "rows_per_page": "200",
```